src/Kovan-DssSpell.sol

```diff
@@ -66,335 +33,18 @@ contract DssSpellAction is DssAction {
-            calc: MCD_CLIP_CALC_PAXUSD_A,
-            hole: 0,
-            chop: 113 * WAD / 100,
-            buf: 105 * RAY / 100,
-            tail: 220 minutes,
-            cusp: 90 * RAY / 100,
-            chip: 1 * WAD / 1000,
-            tip: 1 * RAD,
-            cut: 999 * RAY / 1000,
-            step: 120 seconds,
-            tolerance: 95 * RAY / 100,
-            clipKey: "MCD_CLIP_PAXUSD_A",
-            calcKey: "MCD_CLIP_CALC_PAXUSD_A",
-            flipKey: "MCD_FLIP_PAXUSD_A"
-        }));
-
-        // ----------------------------------- GUSD-A -----------------------------------
-        flipperToClipper(Collateral({
-            ilk: "GUSD-A",
-            vat: MCD_VAT,
-            vow: MCD_VOW,
-            spotter: MCD_SPOT,
-            cat: MCD_CAT,
-            dog: MCD_DOG,
-            end: MCD_END,
-            esm: MCD_ESM,
-            flipperMom: FLIPPER_MOM,
-            clipperMom: CLIPPER_MOM,
-            ilkRegistry: ILK_REGISTRY,
-            pip: address(0),
-            clipper: MCD_CLIP_GUSD_A,
-            flipper: DssExecLib.getChangelogAddress("MCD_FLIP_GUSD_A"),
-            calc: MCD_CLIP_CALC_GUSD_A,
-            hole: 0,
-            chop: 113 * WAD / 100,
-            buf: 105 * RAY / 100,
-            tail: 220 minutes,
-            cusp: 90 * RAY / 100,
-            chip: 1 * WAD / 1000,
-            tip: 1 * RAD,
-            cut: 999 * RAY / 1000,
-            step: 120 seconds,
-            tolerance: 95 * RAY / 100,
-            clipKey: "MCD_CLIP_GUSD_A",
-            calcKey: "MCD_CLIP_CALC_GUSD_A",
-            flipKey: "MCD_FLIP_GUSD_A"
-        }));
-
-        // ----------------------------------- PSM-USDC-A -----------------------------------
-        flipperToClipper(Collateral({
-            ilk: "PSM-USDC-A",
-            vat: MCD_VAT,
-            vow: MCD_VOW,
-            spotter: MCD_SPOT,
-            cat: MCD_CAT,
-            dog: MCD_DOG,
-            end: MCD_END,
-            esm: MCD_ESM,
-            flipperMom: FLIPPER_MOM,
-            clipperMom: CLIPPER_MOM,
-            ilkRegistry: ILK_REGISTRY,
-            pip: address(0),
-            clipper: MCD_CLIP_PSM_USDC_A,
-            flipper: DssExecLib.getChangelogAddress("MCD_FLIP_PSM_USDC_A"),
-            calc: MCD_CLIP_CALC_PSM_USDC_A,
-            hole: 0,
-            chop: 113 * WAD / 100,
-            buf: 105 * RAY / 100,
-            tail: 220 minutes,
-            cusp: 90 * RAY / 100,
-            chip: 1 * WAD / 1000,
-            tip: 1 * RAD,
-            cut: 999 * RAY / 1000,
-            step: 120 seconds,
-            tolerance: 95 * RAY / 100,
-            clipKey: "MCD_CLIP_PSM_USDC_A",
-            calcKey: "MCD_CLIP_CALC_PSM_USDC_A",
-            flipKey: "MCD_FLIP_PSM_USDC_A"
-        }));
+        // ---------------------------- Add MCD_FLASH ----------------------------
+        address MCD_VAT = DssExecLib.vat();
+        Fileable(MCD_FLASH).file("max", 500_000_000 * WAD);
```

Comment:
I'd use `MCD_DAI_FLASH` just in case governance decides in the future to have a `MKR` one that would be totally possible.

Reply:
The design of flash minting contracts (according to the ERC spec) is actually to provide flash mints / loans of all tokens available. This is why there is a `token` address param. It's possible we add flash loan support in the future not only to MKR, but also any gems locked in gem join contracts. In this case we would probably update the flash mint contract to support multiple tokens and retain the MCD_FLASH name imo.

Reply:
Yeah it makes sense